backend/utils.py

```diff
@@ -788,6 +788,13 @@ def set_settings(client_principal, temperature, model, font_family, font_size):
         logging.error(f"[util__module] set_settings: invalid model value {model}.")
         return
 
+    # validate detail level
```

Comment:
@LuisMRL1997 If the detail_level is not valid, shouldn’t the user receive a proper response (e.g., 401 Invalid detail_level)?

To handle this with an early return, wouldn’t it make sense to perform the validation in the endpoint before calling the setSetting function?

You might also want to take a look at this post on the return early pattern: https://medium.com/swlh/return-early-pattern-3d18a41bba8